dosma/core/med_volume.py

```diff
@@ -637,7 +643,50 @@ def to_torch(
             tensor = tensor.contiguous()
         return tensor
 
-    def headers(self, flatten=False):
+    def to_zarr(
+        self,
+        headers_attr: Optional[str] = None,
+        mode: str = "w-",
+        chunks=None,
+        **kwargs,
+    ):
+        """Converts to Zarr store.
+
+        Args:
+            headers_attr (str, optional): Attribute key to store the headers of the `MedicalVolume`
+                in. If `None`, headers will not be saved.
+            mode ({'r', 'r+', 'a', 'w', 'w-'}, optional): Persistence mode: 'r' means read only
+                (must exist); 'r+' means read/write (must exist); 'a' means read/write (create if
+                doesn't exist); 'w' means create (overwrite if exists); 'w-' means create (fail if
+                exists). Default is _w-_ for safety reasons.
+            **kwargs: Additional parameters are passed through to `zarr.creation.open_array`.
+
+        Returns:
+            zarr.Array
+
+        Examples:
+            >>> mv = MedicalVolume(np.ones((10, 20, 30)), np.eye(4))
+            >>> store = zarr.ZipStore("/path/to/store")
```

Comment:
zarr does not require the user to specify a store, so we shouldn't depend on that either

Reply:
> zarr does not require the user to specify a store, so we shouldn't depend on that either

In this case, we would need a `store`, because else we wouldn't be able to instantiate the `MedicalVolume`.

Reply:
sgtm